codegen/smithy-kotlin-codegen/src/main/kotlin/software/amazon/smithy/kotlin/codegen/rendering/auth/SigV4AsymmetricAuthSchemeIntegration.kt

```diff
@@ -25,15 +25,15 @@ import software.amazon.smithy.model.shapes.ShapeId
  * Register support for the `aws.auth#sigv4a` auth scheme.
  */
 class SigV4AsymmetricAuthSchemeIntegration : KotlinIntegration {
-    // Needs to happen after the `SigV4AsymmetricTraitCustomization` (-60).
-    override val order: Byte = -50
-
-    // Needs to be true due to the way integrations are filtered out before application and sigV4a customization.
-    // See 'CodegenVisitor' & 'SigV4AsymmetricTraitCustomization'
-    override fun enabledForService(model: Model, settings: KotlinSettings): Boolean = true
```

Comment:
note: There is one more of these hardcoded `true` in `UnsupportedSigningAlgorithmIntegration`, do you plan to change that one separately?

Reply:
Yes, that one is in `aws-sdk-kotlin`. The change is in a pull request here: https://github.com/awslabs/aws-sdk-kotlin/pull/1245